pom.xml

```diff
@@ -3243,6 +3253,54 @@
           </dependency>
         </dependencies>
       </dependencyManagement>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>net.alchim31.maven</groupId>
+              <artifactId>scala-maven-plugin</artifactId>
+              <configuration combine.self="override">
```

Comment:
do you need combine.self="override" here or on just the children that are overridden?
(But yes this is what I was getting at)

Reply:
Yes, we need `combine.self="override"` here(`merge` is default value),  `merge` and `default` will introduce additional configuration of `compilerPlugins`, but `com.github.ghik:silencer-plugin_2.13.4:1.6.0` not exists  in Maven central repository.

Or we can introduce `com.github.ghik:silencer-plugin_2.13.4:1.7.1` manually and remove `combine.self="override"`, although we don't really need i

Reply:
What about combine.self="override" on compilerPlugins, and leave it empty? does that not effectively remove the config in 2.13?

Reply:
Address fcad7a4 move  combine.self="override" on compilerPlugins